src/containers/Secrets/Secrets.js

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { InlineNotification } from 'carbon-components-react';
+import Modal from '../SecretsModal';
+import DeleteModal from '../../components/SecretsDeleteModal';
+import Table from '../../components/SecretsTable';
+import {
+  clearNotification,
+  deleteSecret,
+  fetchSecrets
+} from '../../actions/secrets';
+import {
+  getSecrets,
+  getSecretsErrorMessage,
+  isFetchingSecrets
+} from '../../reducers';
+
+export /* istanbul ignore next */ class Secrets extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      openNewSecret: false,
+      openDeleteSecret: false,
+      toBeDeleted: ''
+    };
+  }
+
+  componentDidMount() {
+    this.props.fetchSecrets();
+  }
+
+  handleNewSecretClick = () => {
+    this.setState(prevState => {
+      return {
+        openNewSecret: !prevState.openNewSecret
+      };
+    });
+  };
+
+  handleDeleteSecretToggle = () => {
+    this.setState({
+      openDeleteSecret: false,
+      toBeDeleted: ''
+    });
+  };
+
+  handleDeleteSecretClick = e => {
```

Comment:
this function should probably receive an object representing the selected secret `{ name, namespace }` as it shouldn't be aware of implementation details (e.g. use of `svg` or particular attributes such as `id`) of the table

Reply:
Looks like this was updated to receive a single string as a param, but then it needs to be split / parsed to extract the required info. Instead of this it should just receive an object containing the required data.